extensions/QCOM/SPV_QCOM_image_processing2.asciidoc

```diff
@@ -0,0 +1,280 @@
+SPV_QCOM_image_processing2
+==========================
+
+Name Strings
+------------
+
+SPV_QCOM_image_processing2
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Jeff Leger, Qualcomm
+- Ruihao Zhang, Qualcomm
+- Wooyoung Kim, Qualcomm
+
+Notice
+------
+
+Copyright (c) 2023 The Khronos Group Inc. Copyright terms at
+http://www.khronos.org/registry/speccopyright.html
+
+Status
+------
+
+- draft
```

Comment:
Minor: Consider updating the status, unless this really is a spec draft.  I think we usually say "complete" or "shipping" or similar for most of the published and final extensions.

Reply:
Thank you, Ben, for the comments.  I fixed the revision as well as the Status.